src/image/src/ImagePreview.tsx

```diff
@@ -335,6 +338,13 @@ export default defineComponent({
           transformStyle +
           (transition ? '' : 'transition: none;')
       }
+      if (props.imgProps?.style) {
+        Object.entries(
+          normalizeStyle([props.imgProps?.style]) as Record<string, string>
+        ).forEach(([k, v]) => {
+          style[k] = v
+        })
```

Comment:
![image](https://user-images.githubusercontent.com/37191683/190913965-bb14b59e-0468-4146-8434-eb81513cd24d.png)

I dont get the issue. Can someone help?

Reply:
Seems style is typed as `Record<number, xxx>`?